src/app/portlets/dot-templates/dot-templates-routing.module.ts

```diff
@@ -28,6 +28,16 @@ const routes: Routes = [
         resolve: {
             template: DotTemplateCreateEditResolver
         }
+    },
+    {
+        path: 'edit/:id/:inode',
```

Comment:
I think the path should be `edit/:id?inode="xxxx"`

Because like this is too confusing for the user, using a queryparam we indicate the user we are passing an `inode`.

And also using `queryparam` we don't need to create a new route.